### DISCUSSION TITLE
Transliteration

### DISCUSSION THREAD

**Discussion by ashtuchkin** [ashtuchkin/iconv-lite#345] [Ideas]
Probably using callback mechanism in #53.

**Comment by Dexus** [comment]
https://github.com/andyhu/node-transliteration

May help.

**Comment by dresende** [comment]
I really want to move to this but can't without transliteration. Is it possible to use the proposed callback mechanism and the suggested module to accomplish this?

**Comment by ashtuchkin** [comment]
what's your use case? why can't you just use https://github.com/andyhu/node-transliteration ?

**Comment by dresende** [comment]
I work for a company that has a platform that I maintain, and I have a module that does XLS imports and looks tables with matching columns. To handle common mistakes we usually transliterate to ascii to remove all diacritics, like "Dēmokratia" to "Demokratia".

This enables the developers I work with to just define the table headers in lowercase and no diacritics, and the module can lookup the table, no matter what case or diacritics the cell has (well, most of..).

If you have another solution than using `iconv` module, I'll gladly switch. I hate it has to compile and just use it for this for a couple of years now.

**Comment by ashtuchkin** [comment]
Interesting, I see. Well, I think this should be covered by the transliteration module I mentioned above, did you have a chance to look at it?

**Comment by dresende** [comment]
I just looked diagonally at `README` but will definitely take a closer look.

**Comment by dresende** [comment]
Nevermind, transliteration is no longer an issue. I will definitely take a better look to migrate from full iconv.
